src/timePicker/TimePicker.tsx

```diff
@@ -88,17 +88,14 @@ export default class TimePicker extends PureComponent<TimePickerProps, TimePicke
     steps: this.generateTimeSteps(this.props),
   }
 
-  componentWillReceiveProps(newProps: TimePickerProps) {
-    const shouldRegenerateTimeSteps =
-      newProps.minuteStep !== this.props.minuteStep ||
-      newProps.timeStart !== this.props.timeStart
-
-    if (shouldRegenerateTimeSteps) {
-      this.setState(state => ({
-        value: state.value < newProps.timeStart ? newProps.timeStart : state.value,
-        steps: this.generateTimeSteps(newProps),
```

Comment:
So the steps can no more be updated ? That's a breaking change ?

Reply:
Not sure it should be in the state however.

Reply:
It was not working and not used. This is a really edge case.